client/lib/feed-lookup/index.js

```diff
@@ -1,42 +1,43 @@
-// External Dependencies
-var isEmpty = require( 'lodash/isEmpty' ),
-	head = require( 'lodash/head' ),
-	LruCache = require( 'lru' );
-
-// Internal Dependencies
-var inflight = require( 'lib/inflight' ),
-	wpcom = require( 'lib/wp' );
+/**
+ * External Dependencies
+ */
+import { isEmpty, head } from 'lodash';
+import LruCache from 'lru';
+
+/**
+ * Internal Dependencies
+ */
+import { isRequestInflight } from 'lib/inflight';
+import wpcom from 'lib/wp';
 
 const cache = new LruCache( 10 );
```

Comment:
afaict this is only used by the feed posts controller to lookup urls like `/read/feeds/mgoblog.com/posts` and redirect them to the relevant feed id